parm/config/config.resources

```diff
@@ -40,87 +40,52 @@ if [ $step = "prep" -o $step = "prepbufr" ]; then
 
 elif [ $step = "anal" ]; then
 
-    export wtime_anal="02:30:00"
+    export wtime_anal="06:00:00"
```

Comment:
Why has this been bumped up to 6 hours? Won't this cause the job to queue longer than necessary?

Reply:
I have the same question as Cory.

Reply:
Reduced to 2 hours.